firefox-ios/Client/Frontend/Strings.swift

```diff
@@ -3897,6 +3897,167 @@ extension String {
                 tableName: "MainMenu",
                 value: "Syncing paused",
                 comment: "On the main menu, at the top, when the user is signed in but there was an error syncing. The description subtitle for the sync error state.")
+
+            public struct AccessibilityLabels {
+                public static let CloseButtonAccessibilityLabel = MZLocalizedString(
+                    key: "MainMenu.Account.AccessibilityLabels.CloseButton.AccessibilityLabel.v132",
+                    tableName: "MainMenu",
+                    value: "Close",
+                    comment: "The accessibility label for the close button in the Main menu.")
+                public static let MainButtonAccessibilityLabel = MZLocalizedString(
+                    key: "MainMenu.Account.AccessibilityLabels.MainButton.AccessibilityLabel.v132",
+                    tableName: "MainMenu",
+                    value: "Sign in to sync passwords, tabs, and more",
+                    comment: "The accessibility label for the sign in button in the Main menu header view.")
+                public static let BackButtonAccessibilityLabel = MZLocalizedString(
+                    key: "MainMenu.Account.AccessibilityLabels.BackButton.AccessibilityLabel.v132",
+                    tableName: "MainMenu",
+                    value: "Back",
+                    comment: "The accessibility label for the back button in the Main menu header navigation view.")
+            }
+        }
+
+        public struct AccessibilityLabels {
+            public static let MainMenuAccessibilityLabel = MZLocalizedString(
+                key: "MainMenu.AccessibilityLabels.MainMenu.AccessibilityLabel.v132",
+                tableName: "MainMenu",
+                value: "Main Menu",
+                comment: "The accessibility label for the Main Menu.")
+            public static let NewTabMenuItemAccessibilityLabel = MZLocalizedString(
```

Comment:
Sorry if I wasn't clear. I mean that each label should be in the correct struct. So, this would be, for the TabsSection that already exists

```
public struct TabsSection {
    public static let NewTab = MZLocalizedString(.....)
    ...
    public struct AccessibilityLabels {
        public static NewTab = MZLocalizedString(
                key: "MainMenu.TabsSection.AccessibilityLabels.NewTab.v132",
                tableName: "MainMenu",
                value: "New tab",
                comment: "The accessibility label for the Main Menu New Tab item.")
    }
}
```

THat way, it's really clearly organized as to what belongs to what, especially we're doing stuff with strings later on.